data/json/items/armor/boots.json

```diff
@@ -2623,7 +2623,7 @@
     "looks_like": "socks",
     "color": "light_gray",
     "material_thickness": 0.3,
-    "flags": [ "VARSIZE", "SKINTIGHT" ],
+    "flags": [ "VARSIZE", "SKINTIGHT", "ALLOWS_TALONS" ],
```

Comment:
So with the new lizard claws we are talking about all the toes having talons not just a single talon sticking through a hole.  I can't see ankle socks holding up for purpose once you've ripped through the entire front of the foot.  I think these mutants would need to settle for leg warmers.